README.md

```diff
@@ -5,3 +5,30 @@
 
 ## Test Coverage
 [![Test Coverage](https://api.codeclimate.com/v1/badges/88c3d9e8784c8fa76af6/test_coverage)](https://codeclimate.com/github/HumanLearning2021/HumanLearningApp/test_coverage)
```

Comment:
While we're at it, we don't need a dedicated heading for each badge, they are designed to go right under the title. see e.g. [this open source project](https://github.com/arrow-kt/arrow#readme)